psi4/driver/qcdb/libmintsbasissetparser.py

```diff
@@ -195,7 +195,11 @@ def parse(self, symbol, dataset):
                     # or:    H_ECP    O_ECP ...     0
                     if atom_ecp.match(line):
                         ecp_msg = """line %5d""" % (lineno)
-                        symbol_to_am = { 0:0, 1:1, 2:2, 3:3, "s":0, "S":0, "p":1, "P":1, "d":2, "D":2, "f":3, "F":3 }
+                        symbol_to_am = {0: 0,   's': 0,   'S': 0,
+                                        1: 1,   'p': 1,   'P': 1,
+                                        2: 2,   'd': 2,   'D': 2,
+                                        3: 3,   'f': 3,   'F': 3,
+                                        4: 4,   'g': 4,   'G': 4}
                         # This is an ECP spec like "KR-ECP    3     28"
```

Comment:
Are we confident that G is high enough?  I had thought that F was sufficient, based on that periodic table thingy, but apparently I misunderstood the exact meaning of the projectors.

Reply:
I searched the existing ecp basis sets, and `g` was the highest I found. I don't mind adding higher.

Reply:
I figured you would have checked.  I'm fine with up to g in that case.  Want me to merge this now?

Reply:
AFAIK/IIRC to build an ECP for an an atom with maximum angular momentum L, projectors up to L+1 are necessary. So, for the lanthanides and actinides you have F electrons, meaning G projectors. Should be fine.

Reply:
Yup, you're right.  I'd missed / forgotten about the `+1` in, *e.g.* eqn. 1 of [this paper](http://dx.doi.org/10.1063/1.4922844).  I was wondering why G electrons were being accounted for when the G projectors were